bindings/jaxrs/src/main/java/org/commonjava/indy/core/bind/jaxrs/ContentAccessHandler.java

```diff
@@ -117,6 +118,7 @@ public Response doCreate( final String packageType, final String type, final Str
                 builderModifier.accept( builder );
             }
             response = builder.build();
+            contentController.generateHttpMetadataHeaders( transfer, request, response );
         }
```

Comment:
@jdcasey I put the HttpExchangeMetadata child class in the moule core (ContentController will call its construction in core), since indy-bindings-jaxrs depends on indy-core, it's not reasonable to locate the StoreHttpExchangeMetadata in binding-jaxs module which will result the loop of dependency.